docs/pages/inboxes/build-inbox.mdx

```diff
@@ -932,7 +932,7 @@ alix.conversations.findConversationByTopic(conversation.topic)
 // Returns a Group
 alix.conversations.findGroup(group.id)
 // Returns a DM
-alix.conversations.findDm(bo.address)
+alix.conversations.findDmbyInboxId(bo.inboxId);
```

Comment:
is byInboxId still OK for helper methods?

Reply:
Yes